json/src/main/java/com/fasterxml/jackson/jaxrs/json/JacksonJaxbJsonProvider.java

```diff
@@ -24,7 +24,7 @@
  */
 @Provider
 @Consumes(MediaType.WILDCARD) // NOTE: required to support "non-standard" JSON variants
-@Produces(MediaType.WILDCARD)
+@Produces({MediaType.APPLICATION_JSON, "text/json", MediaType.WILDCARD})
```

Comment:
qq: Is there no constant for "text/json" in `MediaType` (I assume not, just verifying)

Reply:
It is not a constant in https://docs.oracle.com/javaee/7/api/javax/ws/rs/core/MediaType.html

Reply:
Interestingly enough "text/xml" is. Oh well.